Sources/MapboxDirections/MBDirectionsOptions.swift

```diff
@@ -382,7 +390,12 @@ open class DirectionsOptions: NSObject, NSSecureCoding, NSCopying {
 
      This property should be set to `MBDirectionsProfileIdentifierAutomobile`, `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`, `MBDirectionsProfileIdentifierCycling`, or `MBDirectionsProfileIdentifierWalking`. The default value of this property is `MBDirectionsProfileIdentifierAutomobile`, which specifies driving directions.
      */
+    #if SWIFT_PACKAGE
+    open var profileIdentifier: MBDirectionsProfileIdentifier
+    #else
```

Comment:
Is this conditional compilation still necessary now that the overall class is annotated with `@objcMembers`?

Reply:
good catch, not needed anymore.